client/rest-high-level/src/main/java/org/elasticsearch/client/license/PutLicenseResponse.java

```diff
@@ -58,7 +55,7 @@
     });
 
     static {
-        PARSER.declareBoolean(constructorArg(), new ParseField("acknowledged"));
```

Comment:
i actually dont mind having these defined here if we dont use them anywhere else.